src/Compilers/CSharp/Test/Symbol/DocumentationComments/ParameterTests.cs

```diff
@@ -833,6 +833,155 @@ class C<T>
             Assert.Equal(expectedValueParameter, model.GetSymbolInfo(names[11]).Symbol);
         }
 
+        [Fact, WorkItem(63758, "https://github.com/dotnet/roslyn/issues/63758")]
+        public void ReadonlyParameter1()
```

Comment:
This may not be the right place for the new tests. It looks like this file is specifically for testing parameter documentation comments.

Reply:
I don't have strong opinion on where exactly the new tests should go. One possibility might be `src\Compilers\CSharp\Test\Symbol\Symbols\Source\ModifierTests.cs` which appears to be a grab bag of a few modifier scenarios.

Reply:
will do.